plugins/modules/vmware_drs_group_manager.py

```diff
@@ -0,0 +1,547 @@
+  state:
+    choices: [ present, absent ]
+    default: present
+    description:
+      - If set to C(present), VMs/hosts will be added to the given DRS group.
+      - If set to C(absent), VMs/hosts will be removed from the given DRS group.
+    type: str
+requirements:
+  - "python >= 2.7"
+  - PyVmomi
+version_added: '1.5.0'
+'''
+
+EXAMPLES = r'''
+---
+- name: Add VMs in an existing DRS VM group
+  delegate_to: localhost
+  community.vmware.vmware_drs_group_manager:
+    hostname: "{{ vcenter_hostname }}"
+    password: "{{ vcenter_password }}"
+    username: "{{ vcenter_username }}"
+    cluster: DC0_C0
+    datacenter: DC0
+    group_name: TEST_VM_01
+    vms:
+      - DC0_C0_RP0_VM0
+      - DC0_C0_RP0_VM1
+    state: present
+
+- name: Add Hosts in an existing DRS Host group
+  delegate_to: localhost
+  community.vmware.vmware_drs_group_manager:
+    hostname: "{{ vcenter_hostname }}"
+    password: "{{ vcenter_password }}"
+    username: "{{ vcenter_username }}"
+    cluster: DC0_C0
+    datacenter: DC0
+    group_name: TEST_HOST_01
+    hosts:
+      - DC0_C0_H0
+      - DC0_C0_H1
+      - DC0_C0_H2
+    state: present
+
+- name: Remove VM from an existing DRS VM group
+  delegate_to: localhost
+  community.vmware.vmware_drs_group_manager:
+    hostname: "{{ vcenter_hostname }}"
+    password: "{{ vcenter_password }}"
+    username: "{{ vcenter_username }}"
+    cluster: DC0_C0
+    datacenter: DC0
+    group_name: TEST_VM_01
+    vms:
+      - DC0_C0_RP0_VM0
+    state: absent
+
+- name: Remove host from an existing DRS Host group
+  delegate_to: localhost
+  community.vmware.vmware_drs_group_manager:
+    hostname: "{{ vcenter_hostname }}"
+    password: "{{ vcenter_password }}"
+    username: "{{ vcenter_username }}"
+    cluster: DC0_C0
+    datacenter: DC0
+    group_name: TEST_HOST_01
+    hosts:
+      - DC0_C0_H0
+    state: absent
+
+'''
+
+RETURN = r'''
+drs_group_member_info:
+    description: Metadata about DRS group
+    returned: always
+    type: dict
+    sample: {
+        "Asia-Cluster1": [
+            {
+                "group_name": "vm_group_002",
+                "type": "vm",
+                "vms": [
+                    "dev-1"
+                ]
+            }
+        ]
+    }
+msg:
+    description: Info message
+    returned: always
+    type: str
+    sample: "Updated host group TEST_HOST_01 successfully"
+'''
+
+try:
+    from pyVmomi import vim
+except ImportError:
+    pass
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.community.vmware.plugins.module_utils.vmware import (
+    PyVmomi,
+    vmware_argument_spec,
+    wait_for_task,
+    find_vm_by_id)
+
+
+class VmwareDrsGroupMemberManager(PyVmomi):
+    """
+    Class to manage DRS group members
+    """
+
+    def __init__(self, module):
+        """
+        Init
+        """
+
+        super(VmwareDrsGroupMemberManager, self).__init__(module)
+
+        self.__datacenter_name = module.params.get('datacenter')
```

Comment:
Why do you use double `_` prefix? It looks like a single `_` would be enough.

See: https://docs.python.org/3/tutorial/classes.html#private-variables